integration-libs/opf/base/opf-api/config/default-opf-api-base-config.ts

```diff
@@ -10,7 +10,7 @@ export const defaultOpfApiBaseConfig: OpfApiConfig = {
   backend: {
     opfApi: {
       endpoints: {
-        getActiveConfigurations: 'active-configurations',
+        getActiveConfigurations: 'v2/active-configurations',
```

Comment:
Should remove path fragment /v2, it was just used for your development and testing.
And the v2 api will be removed from our side once your code was deployed.

Reply:
Thank you, it is removed.